Frontend/src/AddUserForm.svelte

```diff
@@ -0,0 +1,88 @@
+<script>
+    let title;
+    let firstName;
+    let lastName;
+    let email
+
+    let location = [];
+    let group = [];
+
+    const handleSubmit = () => {
+        console.log("Submitted")
+    }
+
+</script>
+
+<main>
+    <form on:submit|preventDefault={handleSubmit} class="modal-div"><!--  prevent default is used to stop the page from refresh -->
+        <label>
+            Title
+            <input type="text" placeholder="" bind:value={title}>
+        </label>
+    
+        <label>
+            First Name
+            <input type="text" placeholder="" bind:value={firstName}>
+        </label>
+        
+        <label>
+            Last Name
+            <input type="text" placeholder="" bind:value={lastName}>
```

Comment:
I don't think you need to define placeholder if you intend to leave these blank. If only some of them will be blank and others won't, it's totally fine to declare it blank to remind you that you did it on purpose.

Reply:
That was a slip. I put them in with the intention of filling them out. I'll put them in!